tools/accuracy_checker/accuracy_checker/adapters/yolo.py

```diff
@@ -1,13 +1,40 @@
-import math
+from collections import namedtuple
 
 import numpy as np
 
 from ..adapters import Adapter
-from ..config import NumberField, StringField, ConfigValidator, ListField, ConfigError
+from ..config import BoolField, NumberField, StringField, ConfigValidator, ListField, ConfigError
 from ..representation import DetectionPrediction
 from ..topology_types import YoloV1Tiny, YoloV2, YoloV2Tiny, YoloV3, YoloV3Tiny
 from ..utils import get_or_parse_value
 
+DetectionBox = namedtuple('DetectionBox', ["x", "y", "w", "h", "confidence", "probabilities"])
+
+
+class YoloOutputProcessor:
+    def __init__(self, coord_correct=lambda x: x, size_correct=lambda x: np.exp(x), conf_correct=lambda x: x,
```

Comment:
I agree with pylint that lambda transition x->x  by default may be excess, probably we can stay None and perform correction only if provided